src/sections/detailBoxes/detailBoxes.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import DetailBox from "./detailBox";
+import styled from "styled-components";
+import tw from "twin.macro";
+
+const BOX_TEXT = [
+    {
+        title: "Connect",
+        topText: "with like-minded students.",
+        bottomText: "with 500+ students from across Canada",
```

Comment:
"Meet like-minded peers, and fellow students from schools across North America. Make lifelong friendships."